docs/source/tracking.rst

```diff
@@ -454,6 +456,8 @@ For example, if you have a Minio server at 1.2.3.4 on port 9000:
 .. code-block:: bash
 
   export MLFLOW_S3_ENDPOINT_URL=http://1.2.3.4:9000
+  
+on both, tracking server and client.
```

Comment:
Why is this change required? All logging APIs will operate at run level and download APIs will return URIs

Reply:
I think this mainly affects `log_artifact` - not sure I understand your comment. 

Since this is the `Server` section I wanted to emphasize that also the client needs this config, even if it has the tracking server (which itself knows the custom endpoint) configured.